src/components/transaction-modal/index.tsx

```diff
@@ -133,7 +133,7 @@ const TransactionModal = ({
     !disabled
 
   return (
-    <Modal title={title} onClose={onClose} {...props}>
+    <Modal title={title} onClose={onClose} {...props} sx={{ width: '500px' }}>
```

Comment:
are all the modals going to be 500px? this is a component prop and this prop is overwritting all of them, remove